packages/config/config/devices/0x0115/v3.json

```diff
@@ -204,6 +204,262 @@
 					"value": 1
 				}
 			]
+		},
+		{
+			"#": "64",
```

Comment:
These labels are going to need to differentiate from each other.
In various UI's they will be provided - and having them all the same won't make for a nice experiance.

```json
{
  "#": "64",
  "label": "Custom parameter 1",
  "valueSize": 2,
  "minValue": -32768,
  "maxValue": 32767,
  "defaultValue": 0
},
{
  "#": "65",
  "label": "Custom parameter 2",
  "valueSize": 2,
  "minValue": -32768,
  "maxValue": 32767,
  "defaultValue": 0
}
```